src/libcephfs.cc

```diff
@@ -852,6 +852,60 @@ extern "C" int ceph_fsetxattr(struct ceph_mount_info *cmount, int fd, const char
 }
 /* end xattr support */
 
+extern "C" void ceph_statx2stat(struct ceph_statx *stx, struct stat *stbuf)
+{
+  stbuf->st_blksize = stx->stx_blksize;
+  stbuf->st_nlink = stx->stx_nlink;
+  stbuf->st_uid = stx->stx_uid;
+  stbuf->st_gid = stx->stx_gid;
+  stbuf->st_mode = stx->stx_mode;
+  stbuf->st_ino = stx->stx_ino;
+  stbuf->st_size = stx->stx_size;
+  stbuf->st_blocks = stx->stx_blocks;
+  stbuf->st_dev = stx->stx_dev;
+  stbuf->st_rdev = stx->stx_rdev;
+  stbuf->st_atim = stx->stx_atime;
+  stbuf->st_ctim = stx->stx_ctime;
+  stbuf->st_mtim = stx->stx_mtime;
+}
+
+
+extern "C" int ceph_stat(struct ceph_mount_info *cmount, const char *path, struct stat *stbuf)
+{
+  int r;
+  struct ceph_statx stx;
+  
+  r = ceph_statx(cmount, path, &stx, CEPH_STATX_ALL_STATS, 0);
+  if (r < 0) 
+    return r;
+  ceph_statx2stat(&stx, stbuf);
+  return r;
+}
+
```

Comment:
I think you probably want to just have this function call Client::stat.

Reply:
ok